protected/humhub/modules/user/components/User.php

```diff
@@ -36,6 +36,11 @@ class User extends \yii\web\User
      */
     protected $permissionManager = null;
 
+    /**
+     * @var string URL to force user to change password
+     */
+    public $mustChangePasswordUrl = '/user/must-change-password';
```

Comment:
I'd rename it to `$mustChangePasswordRoute` since its not an url

Reply:
Ok, I renamed that to `$mustChangePasswordRoute`, but please note I named this as `...Url' because here https://github.com/humhub/humhub/blob/enh/3996-force-password-change/protected/humhub/config/web.php#L36 we have `'loginUrl' => ['/user/auth/login'],` as well.

Reply:
You are right its passed to ` Yii::$app->getResponse()->redirect` which also supports real urls same with `loginUrl`.